src/webgpu/api/validation/buffer/create.spec.ts

```diff
@@ -26,21 +28,56 @@ g.test('size')
         kBufferSizeAlignment * 2,
       ])
   )
-  .unimplemented();
+  .fn(t => {
+    const { mappedAtCreation, size } = t.params;
+    const isValid = !mappedAtCreation || size % kBufferSizeAlignment === 0;
+    const usage = BufferUsage.COPY_SRC;
```

Comment:
nit: Since this is inside a test it can use GPUBufferUsage
```suggestion
    const usage = GPUBufferUsage.COPY_SRC;
```

Reply:
Done.